sdks/python/apache_beam/transforms/external.py

```diff
@@ -414,7 +414,7 @@ def __init__(
 
   def expand(self, pcolls):
     # Expand the transform using the expansion service.
-    return pcolls | ExternalTransform(
+    return pcolls | self._payload_builder._identifier >> ExternalTransform(
```

Comment:
Do possible characters and format for payload builder identifier and PTransform labels match ?

Reply:
I can't find documentation on what the standard is for PTransform labels, but I tried the Storage API xlang wrapper with URN `beam:schematransform:org.apache.beam:bigquery_storage_write:v2` and it shows up fine on the Dataflow DAG: 
![image](https://github.com/apache/beam/assets/65791736/52121e1c-c51d-41c4-9fda-280757b67813)

Reply:
Also, I think Dataflow might fail if you use the same label twice (i.e. if you use that same SchemaTransform twice in a pipeline). Can you check that ?

Reply:
I tried the following on Dataflow and it worked fine:
with beam.Pipeline(options=opts) as p:
  input = p | beam.Create([{"hi": "hello"}])
  input | "write1" >> beam.io.WriteToBigQuery(table="...1",
                             method="STORAGE_WRITE_API",
                             schema="hi:STRING")
  input | "write2" >> beam.io.WriteToBigQuery(table="...2",
                                  method="STORAGE_WRITE_API",
                                  schema="hi:STRING")

I think it just cares about the top-level transform naming. Our IOs and other composites have consistent naming and it's up to the user to label top level transforms where needed. Without the top-level labels, I get the following error:
`A transform with label "WriteToBigQuery" already exists in the pipeline. To apply a transform with a specified label write pvalue | "label" >> transform`